Source/Workers/decodeDraco.js

```diff
@@ -82,6 +84,21 @@ define([
                         componentDatatype : ComponentDatatype.fromTypedArray(vertexArray)
                     }
                 };
+
+                transform = new draco.AttributeQuantizationTransform();
+                if (transform.InitFromAttribute(attribute)) {
+                    var minValues = new Float32Array(numComponents);
```

Comment:
It doesn't really matter, but could this be a plain Array instead? Since the array is pretty small using a `Float32Array` almost seems misleading.